index.php

```diff
@@ -0,0 +1,2 @@
+<?php
+header('Location: index.html');
```

Comment:
Not as far as I know. Static files don't have a build pack - although, I could just put it on GitHub pages for now, might be easier :grin: .

Regarding PHP closing tags, yeah the standard now (well for a while) is not to include the closing PHP tag